### PR TITLE
pr_checker: add warn about conflicts

### PR DESCRIPTION
**Pull request by alopezsanchez**
Question:
- Should we log something when `mergeable === 'UNKNOWN'`?

Please, review it, I don't know if I put all the things in the correct place.

demo

Fixes:  https://github.com/joyeecheung/node-core-utils/issues/68

### CODE REVIEW

**Comment by cPhost** [comment]
@alopezsanchez you can log something like
`
[WARN] GitHub is calculating Mergability
`

**Comment by joyeecheung** [comment]
I wouldn't think it's necessary to warn about uncertain conflicts, people will always find out when they push, let's keep the volume to the minimum.

**Comment by alopezsanchez** [comment]
Okay. Give me some hours to revert the last commit.

**Comment by alopezsanchez** [comment]
Done.

File: lib/pr_checker.js [hunk 1]
```diff
@@ -268,6 +272,20 @@ class PRChecker {
...
+
+  checkMergeableState() {
+    const {
+      pr, logger
+    } = this;
+
+    let status = true;
+    if (pr.mergeable && pr.mergeable === CONFLICTING) {
+      logger.warn('This PR has conflicts that must be resolved');
+      status = false;
```
Comment (apapirovski) [inline lib/pr_checker.js]: Let's just simplify this to `return false;`

File: lib/pr_checker.js [hunk 2]
```diff
@@ -268,6 +272,20 @@ class PRChecker {
...
+    }
+
+    return status;
```
Comment (apapirovski) [inline lib/pr_checker.js]: This can then be `return true;` and we can remove the `let status = true;`

**Comment by cPhost** [comment]
Merging it now, since i think everybody approved it.

**Pull request merged**